src/query/storages/fuse/src/io/read/block/parquet/deserialize.rs

```diff
@@ -27,16 +29,31 @@ use parquet::basic::Compression as ParquetCompression;
 
 use crate::io::read::block::block_reader_merge_io::DataItem;
 use crate::io::read::block::parquet::adapter::RowGroupImplBuilder;
+use crate::io::read::block::parquet::arrow_parquet::arrow_to_parquet_schema_fix;
 
 /// The returned record batch contains all deserialized columns in the same nested structure as the original schema.
 pub fn column_chunks_to_record_batch(
     original_schema: &TableSchema,
     num_rows: usize,
     column_chunks: &HashMap<ColumnId, DataItem>,
     compression: &Compression,
+    block_path: &str,
 ) -> databend_common_exception::Result<RecordBatch> {
+    let use_v1 = !block_path.contains("_b/g")
```

Comment:
The prefix of block_path might contain 'g'. The function `try_extract_uuid_str_from_path()` might be helpful.